app/src/main/res/layout/sheet_custom_drug_entry.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:id="@+id/rootLayout"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:background="@color/white">
+
+  <TextView
+    android:id="@+id/titleTextView"
+    android:layout_width="0dp"
+    android:layout_height="wrap_content"
+    android:layout_alignParentStart="true"
+    android:layout_centerVertical="true"
+    android:layout_marginStart="@dimen/spacing_24"
+    android:layout_marginTop="24dp"
+    android:layout_marginEnd="@dimen/spacing_16"
+    android:layout_toStartOf="@+id/removeButton"
+    android:ellipsize="end"
+    android:maxLines="2"
+    android:textAppearance="?attr/textAppearanceHeadline6"
+    android:textColor="?attr/colorOnBackground"
+    app:layout_constraintEnd_toStartOf="@+id/removeButton"
+    app:layout_constraintHorizontal_bias="0.0"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:text="Procardia XL" />
+
+  <com.google.android.material.button.MaterialButton
+    android:id="@+id/removeButton"
+    style="@style/Widget.Simple.Button.TextButton.Red1"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_alignParentEnd="true"
+    android:layout_gravity="center_vertical"
+    android:layout_marginTop="24dp"
+    android:layout_marginEnd="24dp"
+    android:minWidth="@dimen/spacing_0"
+    android:minHeight="@dimen/spacing_0"
+    android:text="@string/custom_drug_entry_sheet_remove"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintTop_toTopOf="parent"
+    tools:visibility="visible" />
+
+  <LinearLayout
```

Comment:
We can probably avoid this nesting and create a chain of these two text fields.

Reply:
Will resolve this in the next PR